frontend/packages/dev-console/src/utils/__tests__/pipeline-augment-test-data.ts

```diff
@@ -94,3 +94,369 @@ export const testData: PipelineAugmentData[] = [
     },
   },
 ];
+
+export enum DataState {
```

Comment:
Conflict resolution problem? This should have been moved to `dev-console/src/test/pipeline-data.ts` in one of my last PRs that were merged yesterday.

Did you add something to the lines below?

Reply:
nopes, I just accepted incoming changes

Reply:
Well, if you didn't change anything, just remove all the "added" lines here. Not sure how you got them back :) (are you on latest?)

Reply:
I verified, and all the code is using the ones from `pipeline-data` so these lines are just dead code.

Reply:
yup, ok. I m removing them from pipeline-data